README.md

```diff
@@ -40,26 +40,9 @@ We have a dedicated stream for this project [#mentorship-system](https://anitab-
 - To save JWT token, [Flutter Secure Storage](https://pub.dev/packages/flutter_secure_storage) is used
 - If you find something in code that looks a bit odd, it might be some useful extension method from `lib/extensions`
 
-#### Setup instructions
+## Setup Instructions
 
-1. Install Flutter by following instructions from [flutter.dev](https://flutter.dev).
-2. Make sure you install the **Flutter** and **Dart** plugins. The installation instructions above should have instructions on this too.
-3. Fork and clone the [mentorship-flutter](https://github.com/anitab-org/mentorship-flutter) repository to your local machine.
-4. Open the project on your IDE. Don't be shocked if you see error warnings.
-5. Open the terminal and run `flutter channel beta` to switch from stable channel to beta channel of flutter. The beta channel is the one we are using for this project.
-6. Once done, run the project by running `flutter run` in the terminal to run the app in debug mode. To build a release build you can do the following:
-
-  *for Android app*
-
-  In the terminal, run the `flutter build apk` command. To build the apk specific to your device arch you can run `flutter build apk --split-per-abi` or `flutter build appbundle --target-platform android-arm,android-arm64,android-x64` (remove arguments which are not required) to get only your arch build. You can read more about this [here](https://flutter.dev/docs/deployment/android)
-
-  *for IOS app*
-
-  To build a release for IOS app, run `flutter build ios` from the terminal. To learn more on creating build archive, release app on TestFlight or to App Store, click [here](https://flutter.dev/docs/deployment/ios).
-
-
-
-By default the backend of this project is set to the mentorship-backend dev server. You are now set to go.
+In order to setup the project locally, setup instructions have been defined here(docs/setup-instructions.md)
```

Comment:
I think @sakshivij you forgot to add link for set-up instructions file,  correct me if i am wrong 🤔 ?

Reply:
This file would be merged into docs folder. I have added as far as I understand. Do you think there is something incorrect here?

Reply:
actually, i didn't find any link to that docs in this readme so, it would be great you can just directly link it here :) @sakshi1499

Reply:
Also, it would be great if you can just resolve the merge conflicts :)

Reply:
@robotjellyzone https://github.com/anitab-org/mentorship-flutter/pull/78/files#diff-120f9fc5414bd9af18a5dcc49c68b760  ?

Reply:
oh !! ok thanks @Techno-Disaster actually i thought that on clicking that link one should be taken to that setup file . but this way is also fine

Reply:
although the brackets here are wrong so its not linked correctly please check this @sakshivij

Reply:
@robotjellyzone quick tip , to check how the readme will look u can check the fork branch [here](https://github.com/sakshivij/mentorship-flutter/tree/feature/Readme)

Reply:
Oh!! thanks @Techno-Disaster now i got it but what i want is just to add a link so that the user can directly navigate !!

Reply:
Working on the changes.